iwyu_ast_util.cc

```diff
@@ -1441,12 +1456,14 @@ bool HasImplicitConversionConstructor(const Type* type) {
   return HasImplicitConversionCtor(cxx_class);
 }
 
-map<const clang::Type*, const clang::Type*>
-GetTplTypeResugarMapForClassNoComponentTypes(const clang::Type* type) {
+TemplateInstantiationData GetTplInstDataForClassNoComponentTypes(
```

Comment:
Same here with `clang::Type` -> `Type`. I think it's a nice change for readability with the additional `function<>` even if it's not strictly necessary.